person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative 'nameable'
 
 class Person < Nameable
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
   attr_reader :id
 
   def initialize(age, parent_permission: true, name: 'unknown')
```

Comment:
- [ ] Please make sure to add a method to add rentals in the person class for the association to work as expected.

```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```